hokusai/production.yml

```diff
@@ -273,7 +288,7 @@ spec:
             periodSeconds: 30
           resources:
             requests:
-              cpu: 200m
+              cpu: 100m
```

Comment:
most of the time [cpu usage](https://app.datadoghq.com/dashboard/2n5-6tr-ypp/kubernetes-cpumemory-usage-by-deployment?from_ts=1616080309689&live=true&tile_size=s&to_ts=1616685109689&tpl_var_deployment=convection-sidekiq&tpl_var_env=production&tpl_var_hpa=convection-sidekiq) is near-zero. Reduce it so less cpu is held idle during low tide. However, there are tall spikes from time to time, so HPA max (above) is raised so that during a spike, aggregate cpu-per-pod X pod-count is still 400m.